crates/viewer/re_viewport/src/test_context_ext.rs

```diff
@@ -56,16 +56,17 @@ impl TestContextExt for TestContext {
             if let Some(setup_blueprint) = setup_blueprint.take() {
                 self.run(egui_ctx, |ctx| {
                     let mut viewport_blueprint =
-                        ViewportBlueprint::from_db(&self.blueprint_store, &self.blueprint_query);
+                        ViewportBlueprint::from_db(ctx.blueprint_db(), &self.blueprint_query);
                     result = Some(setup_blueprint(ctx, &mut viewport_blueprint));
                     viewport_blueprint.save_to_blueprint_store(ctx);
                 });
 
                 self.handle_system_commands();
 
                 // Reload the blueprint store and execute all view queries.
+                let blueprint_query = self.blueprint_query.clone();
                 let viewport_blueprint =
-                    ViewportBlueprint::from_db(&self.blueprint_store, &self.blueprint_query);
+                    ViewportBlueprint::from_db(self.active_blueprint(), &blueprint_query);
```

Comment:
it feels slightly off that the blueprint would be accessed by two different means here.

Reply:
ah sorry, I overlooked that fact that the first one is in a closure. nvm then.